<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Myriad Pro</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Project Thermite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/ICONS/Icon.png</normaloff>:/ICONS/Icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CodeEditor" name="editor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1251</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="Terminal" name="terminal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>1251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionAbout"/>
    <addaction name="actionReadme"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionAssemble"/>
    <addaction name="actionSimulate"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/New.png</normaloff>:/ICONS/New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a C++ code file</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Open.png</normaloff>:/ICONS/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a C++ code file</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/SelectAll.png</normaloff>:/ICONS/SelectAll.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select all text</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Undo.png</normaloff>:/ICONS/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Redo.png</normaloff>:/ICONS/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/About.png</normaloff>:/ICONS/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Project Thermite</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionReadme">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Document.png</normaloff>:/ICONS/Document.png</iconset>
   </property>
   <property name="text">
    <string>Readme</string>
   </property>
   <property name="toolTip">
    <string>View readme</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Save.png</normaloff>:/ICONS/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the file</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Settings.png</normaloff>:/ICONS/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change the settings</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Compile.png</normaloff>:/ICONS/Compile.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile (C++ Code)</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAssemble">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Assemble.png</normaloff>:/ICONS/Assemble.png</iconset>
   </property>
   <property name="text">
    <string>Assemble</string>
   </property>
   <property name="toolTip">
    <string>Assemble (MIPS Assembly Language)</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Simulate.png</normaloff>:/ICONS/Simulate.png</iconset>
   </property>
   <property name="text">
    <string>Simulate</string>
   </property>
   <property name="toolTip">
    <string>Simulate (MIPS Machine Language)</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Run.png</normaloff>:/ICONS/Run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run (C++ Code)</string>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/SaveAs.png</normaloff>:/ICONS/SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Copy.png</normaloff>:/ICONS/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Cut.png</normaloff>:/ICONS/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ICONS/Paste.png</normaloff>:/ICONS/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>Terminal</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">terminal.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
